types/foundry/client-esm/canvas/regions/mesh.d.ts

```diff
@@ -0,0 +1,55 @@
+import type { Region } from "../../../client/pixi/placeables/region.d.ts";
+
+/** A mesh of a {@link Region}. */
+export class RegionMesh<TShader extends AbstractBaseShader = RegionShader> extends PIXI.Container {
+    /**
+     * Create a RegionMesh.
+     * @param region           The Region to create the RegionMesh from.
+     * @param [shaderClass]    The shader class to use.
+     */
+    constructor(region: Region, shaderClass?: TShader);
+
+    /** The Region of this RegionMesh. */
+    get region(): Region;
+
+    /** The shader bound to this RegionMesh. */
+    get shader(): TShader;
+
+    /** The blend mode assigned to this RegionMesh.*/
+    get blendMode(): (typeof PIXI.BLEND_MODES)[keyof typeof PIXI.BLEND_MODES];
+    set blendMode(value: (typeof PIXI.BLEND_MODES)[keyof typeof PIXI.BLEND_MODES]);
+
+    /**
+     * The tint applied to the mesh. This is a hex value.
+     *
+     * A value of 0xFFFFFF will remove any tint effect.
+     * @defaultValue 0xFFFFFF
+     */
+    get tint(): number;
+    set tint(tint: number);
+
+    /** The tint applied to the mesh. This is a hex value. A value of 0xFFFFFF will remove any tint effect. */
+    protected _tintColor: PIXI.Color;
+
+    /**
+     * Cached tint value for the shader uniforms.
+     * @type {[red: number, green: number, blue: number, alpha: number]}
```

Comment:
This looks odd being here given we have typescript, but is the type of _cachedTint then [number, number, number, number] instead of number[] ?

Reply:
Oh true, the type is wrong. It should be [number, number, number, number].

Reply:
I've changed the `@type` to `@returns`. I'm not sure what our stance on editorializing the docs is.